pkg/http_endpoint/v1/router.go

```diff
@@ -1,16 +1,21 @@
 package v1Api
 
 import (
+	"encoding/json"
 	"fmt"
 	"github.com/pressly/chi"
 	"net/http"
+	"strconv"
+
+	"visualization-api/pkg/grafanaclient"
 	"visualization-api/pkg/http_endpoint/authentication"
 	"visualization-api/pkg/http_endpoint/common"
 	"visualization-api/pkg/logging"
 )
 
 const adminAPIPrefix = "/admin"
 const authPrefix = "/auth"
+var gSession *grafanaclient.Session
```

Comment:
Correct way of passing grafanaClient is not using global variable, but wrapping in additional function.